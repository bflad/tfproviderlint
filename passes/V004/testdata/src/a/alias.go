package a

import (
	v "github.com/hashicorp/terraform-plugin-sdk/helper/validation"
)

func falias() {
	// Passing

	_ = v.IsIPAddress

	// Comment ignored

	//lintignore:V004
	v.SingleIP()

	v.SingleIP() //lintignore:V004

	// Failing

	_ = v.SingleIP // want "deprecated v.SingleIP should be replaced with v.IsIPAddress"
	v.SingleIP()   // want "deprecated v.SingleIP() should be replaced with v.IsIPAddress"
}
