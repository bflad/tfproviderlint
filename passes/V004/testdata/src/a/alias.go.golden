package a

import (
	v "github.com/hashicorp/terraform-plugin-sdk/helper/validation"
)

func falias() {
	// Passing

	_ = v.IsIPAddress

	// Comment ignored

	//lintignore:V004
	v.SingleIP()

	v.SingleIP() //lintignore:V004

	// Failing

	_ = v.IsIPAddress // want "deprecated v.SingleIP should be replaced with v.IsIPAddress"
	v.IsIPAddress     // want "deprecated v.SingleIP() should be replaced with v.IsIPAddress"
}
